docs/src/app/components/pages/components/lists.jsx

```diff
@@ -35,10 +35,10 @@ let CodeExample = require('../../code-example/code-example');
 
 
 
-class ListsPage extends React.Component {
+export default class ListsPage extends React.Component {
 
-  constructor() {
-    super();
+  constructor(props) {
```

Comment:
I don't know of any reason to NOT call props? I can remove if you don't like it, but it seemed like having it there was better than not.